readme.md

```diff
@@ -16,6 +16,7 @@ One CRUD operation to manage Companies - create/edit/update/delete.
 - Run __php artisan key:generate__
 - Run __php artisan migrate__
 - Run __php artisan passport:install__
+- Run __composer update__
```

Comment:
@Sylk how would that composer update help if there was composer install done just before it? Doesn't make sense, sorry.

Reply:
I ran the commands in this, and it had trouble until I ran composer update after passport install.

As to why, well I'm just referencing #12 because I had the same issue and that resolved it for me.